### ISSUE TITLE
Crash when ASC imported into unoriented samples

### ISSUE THREAD

**Issue by pont-us** [pont-us/PuffinPlot#325]
Report from Kuo-Hang Chen, 2018-11-27. Steps to reproduce:

1. Read a file using a custom format with no sample or formation    orientation. All correction orientations/parameters will be NaN.

2. Import AMS data from an ASC file with one or more sample names    that match sample names already in the suite.

Result: PuffinPlot crashes in the Tensor constructor, which throws an IllegalArgumentException when it finds that the correction matrices contain NaNs -- because they have been constructed from the sample and formation orientations, which were all left as NaN during the initial custom file import.

Desired behaviour:

1. Read a file using a custom format with no sample or formation    orientation. File importer sets reasonable default values for    the orientation parameters (the usual 0/90 and 0/0).

2. Import AMS data from an ASC file with one or more sample names    that match sample names already in the suite. AMS importer asks    whether to overwrite existing orientation parameters with    values from the ASC file and acts accordingly.

A simpler fix would also be possible: retain the behaviour of leaving the corrections as NaN, and have the ASC importer overwrite them automatically in this case. However, I feel that having NaN corrections is in itself a bug, even if this is the only crash that it's so far led to. The correction parameters are so widely used that it's safer to have them well-defined from the start. Graceful handling of NaN values would in any case probably just consist of falling back to the same sensible defaults, so it makes more sense to set them explicitly from the start.

**Comment by pont-us** [comment]
```
Bugs Everywhere data:
Created at: 2018-12-01 10:12:11
Status: open
Severity: minor
UUID: 2588eaed-fd41-4e3b-a0e2-ab34e2769c3c
Short name: a88/258
```

**Comment by pont-us** [comment]
Fixed in c5e73a8ea9a6082.
